subsys/tfm/boards/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2020, Nordic Semiconductor ASA.
```

Comment:
2021?

Reply:
I'm actually not sure, since this is written in 2020 (or is it? :laughing: )

Reply:
will change

Reply:
Changed